### DISCUSSION TITLE
Add LoadAsync(string filename)

### DISCUSSION THREAD

**Discussion by pekspro** [SixLabors/ImageSharp#1255] [Ideas] [answered]
You could load images by:

    Image.Load(string path);

But there is no:

    Image.LoadAsync(string path);

There are other `LoadAsync` methods, but they all requires a `Stream`. Not hard to create a `FileStream`, but I it would nice is this a built in from start :-)

The same is also true for the `SaveAsync` methods.

**Comment by JimBobSquarePants** [comment, marked as answer]
Sounds reasonable. Feel free to open a PR to add the functionality

**Reply by pekspro** [reply]
I created PR #1256 that adds several extensions for saving files, and PR #1257 for LoadAsync methods,
